docs/view_components/create_scenario.rst

```diff
@@ -0,0 +1,54 @@
+..
+    one can then cross link to this view component by using
+    :ref:`create_scenario-label`
+
+Create Scenario
+---------------
+
+Create scenario view-component enables the user to create a new scenario object where the energy system can be implemented.
+
+Attributes
+^^^^^^^^^^
+
+**Combination of a text field and other GUI elements**
+
+    Properties:
+        * Allows user to navigate through the file system either graphically or through text inputs
+        * Allows user to name the scenario object and write it to a chosen location in the computer's file system
+
+    Requirements index: 1
+
+Actions
+^^^^^^^
+
+1. When a new scenario is created, it should appear in the the list of available scenarios within the project in the scenarios view
```

Comment:
This is a requirement, an action is "What can I do when I see this view-component", "How can I *act* on it"